curvefs/conf/tools.conf

```diff
@@ -10,3 +10,19 @@ topoFilePath=curvefs/test/tools/topo_example.json  # __CURVEADM_TEMPLATE__ /usr/
 metaserverAddr=127.0.0.1:6701  # __CURVEADM_TEMPLATE__ ${cluster_metaserver_addr} __CURVEADM_TEMPLATE__  __ANSIBLE_TEMPLATE__ {{ groups.metaserver | join_peer(hostvars, "metaserver_listen_port") }} __ANSIBLE_TEMPLATE__
 # etcd
 etcdAddr=127.0.0.1:12379  # __CURVEADM_TEMPLATE__ ${cluster_etcd_addr} __CURVEADM_TEMPLATE__  __ANSIBLE_TEMPLATE__ {{ groups.etcd | join_peer(hostvars, "etcd_listen_client_port") }} __ANSIBLE_TEMPLATE__
+# fs
+blockSize=1048576
+fsType=s3
+# volume
+volumeSize=1048576
+volumeBlockSize=4096
+volumeName=volume
+volumeUser=user
+volumePassword=password
+# s3
+s3Ak=ak
+s3Sk=sk
+s3Endpoint=endpoint
+s3BucketName=bucket
+s3BlockSize=1048576
```

Comment:
Need use '__CURVEADM_TEMPLATE__' ?

Reply:
> Need use '**CURVEADM_TEMPLATE**' ?

this [pr](https://github.com/opencurve/curveadm/pull/33 ) will fix it